src/envoy/mixer/http_filter.cc

```diff
@@ -64,13 +67,38 @@ const LowerCaseString kIstioAttributeHeader("x-istio-attributes");
 // Referer header
 const LowerCaseString kRefererHeaderKey("referer");
 
+std::shared_ptr<Auth::IssuerInfo> CreateIssuer(const JWT& jwt) {
+  auto issuer = std::make_shared<Auth::IssuerInfo>();
+  issuer->name_ = jwt.issuer();
```

Comment:
Those fields are supposed to be private, can you move this piece of code to the constructor of IssuerInfo?

i.e. IssuerInfo(name, pkey_type, uri_, audiences, cluster)

Reply:
Done